org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorServletListener.mtl

```diff
@@ -99,13 +96,19 @@ public class [javaClassNameForAdaptorServletListener(anAdaptorInterface) /] impl
     @Override
     public void contextInitialized(final ServletContextEvent servletContextEvent)
     {
+        //These are default values. You can modify any of them early in this method.
+        String basePathEnvKey = "LYO_BASE";
```

Comment:
We can actually but them right into the protected block and save the trouble of overriding them just a few lines below :)

Reply:
True, but can't be bothered for now ;.)